### PR TITLE
Run rpath check in parallel

### PR DESCRIPTION
**Pull request by ffesti**
See #1713

### CODE REVIEW

**Comment by stratakis** [comment]
Proposed also from the mailing list to speedup things:

-    check_rpath $i "RPATH"
-    check_rpath $i "RUNPATH"
+    check_rpath $i "RPATH|RUNPATH"

**Comment by ffesti** [comment]
Yeah, I figured this would help, but it took your message for me to see how to do that in an clean and easy way.

Can someone test the changes, please. May be even with one of the long running builds like kernel.

File: scripts/check-rpaths-worker [hunk 1]
```diff
@@ -105,6 +104,7 @@ function check_rpath() {
 
     allow_ORIGIN=1
     for j; do
+	lower=$(echo $j | grep -o "$2" | awk '{print tolower($0)}')
```
Comment (fweimer-rh) [inline scripts/check-rpaths-worker:107]: I think you can use the `,,` expansion operator here:

```
$ bash -c 'echo ${0,,*}' FOO
foo
```

**Comment by stratakis** [comment]
> Yeah, I figured this would help, but it took your message for me to see how to do that in an clean and easy way.
> 
> Can someone test the changes, please. May be even with one of the long running builds like kernel.

I'm doing some local initial testing.

**Comment by stratakis** [comment]
kernel result on koji with the current version on rawhide:

```
real	13m51.517s
user	8m53.216s
sys	7m34.105s
```

kernel results locally with the patch:
```
real	1m10.202s
user	8m19.161s
sys	5m6.297s
```

For completion I'll also try it locally without the patch.

**Comment by stratakis** [comment]
Updated the comment. It seems the optimizations do indeed work from my local testing.

**Comment by ffesti** [comment]
Ok, there were still some issues with quoting. Should work now and produce proper messages.

**Pull request merged**